inc/clone-amend/namespace.php

```diff
@@ -56,6 +56,11 @@ function set_enabled_post_types( array $enabled_post_types ) : array {
 	$public_post_types = get_post_types( [ 'public' => true ], 'names' );
 	$post_types = Altis\get_config()['modules']['workflow']['clone-amend']['post-types'] ?? null;
 
+	// If WooCommerce exsists, unset the post type Product.
+	if ( class_exists( 'woocommerce' ) ) {
```

Comment:
Although PHP is case-insensitive the core WC plugin uses `WooCommerce` capitalisation. I think we should keep it consistent

Reply:
```suggestion
	if ( class_exists( 'WooCommerce' ) ) {
```

Reply:
I used this as a reference where it was originally all lowercase by WooCommerce : https://woocommerce.com/document/query-whether-woocommerce-is-activated/

Based on the above docs, should I maybe include the `is_woocommerce_activated` check too?

Reply:
Ah ok, well no worries then, that code example is pretty bad though! If you see an if statement that straight up returns `true` or `false` you can just return the condition in the `if` e.g. `return class_exists( 'WooCommerce' )`.

We don't need the function for a one-time thing like this.